README.md

```diff
@@ -1 +1,23 @@
-# hw01
\ No newline at end of file
+# **Samuel Northrup - HW01 - MACS 30500**
+
+## *Biographical Information*
+I grew up in rural Kentucky, and became interested in the Middle East after studying Arabic in Amman, Jordan. I loved seeing and learning about different cultures and worldviews. I spent the rest of my time in high school and college learning Arabic so that one day I could return to the region and explore the culture even more. 
+
+## *Education and Work*
+I studied political science and international economics at the University of Kentucky. I then worked at The Washington Institute, researching military affairs in the Middle East. I decided to joing the Master's in Middle East Studies program at UChicago because of its emphasis on regional history and its flexibility in reserach topic and design. 
+
+## *Research Interests*
+I study healthcare systems and humanitarian aid programs in the Middle East. Specifically, I research the history of the Syrian healthcare system from French colonial occupation in 1920 to the present day. I became interested in this topic since reading an [article](https://www.nybooks.com/articles/2014/02/20/syrias-polio-epidemic-suppressed-truth/) about the outbreak of poliovirus in Eastern Syria. I believe that by understanding the *political* dimensions of healthcare, the international community can better provide medical care to vulnerable populations in times of peace *and* conflict. 
+
+I want to understand several different questions about healthcare in the years leading up to the Syrian Civil War:
+
+ * Did families sympathetic to the regime have more access to healthcare facilities?
+ * Did the regime provide some hospitals with better equipment and funding than others? Why? 
+ * Why the regime both outlaw the provision of any healthcare to the opposition while promising universal healthcare?
+
+
+
```

Comment:
Overall good work! You completed all the requirements (title, list, link, italics) and demonstrated competence with R!